onnxruntime/test/util/compare_mlvalue.cc

```diff
@@ -364,8 +364,9 @@ std::pair<COMPARE_RESULT, std::string> VerifyValueInfo(const ONNX_NAMESPACE::Val
     }
     std::vector<int64_t> shape = GetTensorShape(info.get());
     if (!AreShapesEqual(shape, t.shape())) {
-      std::string result;
-      if (!google::protobuf::TextFormat::PrintToString(t.shape(), &result)) {
+      std::string result = t.shape().SerializeAsString();
```

Comment:
Is this code correct?

Reply:
Do you see anything wrong? This is how pytorch and onnx get around with the unavailability of PrintToString. See the following:

https://github.com/pytorch/pytorch/pull/14150/files
https://github.com/onnx/onnx/blob/e4e3d46c0bb928882cb38f37289193c82f462bee/onnx/proto_utils.h#L18
https://github.com/pytorch/pytorch/blob/master/caffe2/utils/proto_utils.cc

Reply:
Is it a text string, or un-readable byte array?

Reply:
https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.message_lite#MessageLite.SerializeAsString.details

Reply:
You can search the error message in the CI build logs.  You code doesn't work as expected.

Reply:
fixed. somehow it doesn't show up in the windows ci logs.